components/sections/Projects/ProjectTitle.jsx

```diff
@@ -0,0 +1,27 @@
+import Image from "next/image";
+
+const ProjectTitle = ({ name, openProjectPage, width, active, index }) => {
+
+  return (
+    <div
+      className="cursor-pointer relative group flex items-start"
+      onClick={width > 1024 ? openProjectPage: () =>{}}
```

Comment:
```suggestion
      onClick={()=>width > MINIMUM_DESKTOP_WIDTH ? openProjectPage (): null }
```

or you can just make `openprojectPage` an inline function